guidelines/index.html

```diff
@@ -1025,7 +1025,7 @@ <h1 id="title" class="title">Web Content Accessibility Guidelines (WCAG) 2.2</h1
 
     	<div class="note" role="note" id="issue-container-generatedID-12"><div role="heading" class="note-title marker" id="h-note-12" aria-level="5"><span>注記 1</span></div><p class="">ユーザエージェントによって制御されている追加コンテンツの例としては、HTML の <a href="https://html.spec.whatwg.org/multipage/dom.html#the-title-attribute"><code>title</code> 属性</a> [<cite><a class="bibref" data-link-type="biblio" href="#bib-html" title="HTML Standard">HTML</a></cite>] を用いて作られているブラウザのツールチップが挙げられる。</p></div>
      <div class="note" role="note" id="issue-container-generatedID-13"><div role="heading" class="note-title marker" id="h-note-13" aria-level="5"><span>注記 2</span></div><p class="">ホバー時やフォーカス時に表示されるカスタムツールチップ、サブメニュー、他の非モーダルポップアップは、この達成基準の適用対象となる「追加コンテンツ」の例である。</p></div>
-     <div class="note" role="note" id="issue-container-generatedID-14"><div role="heading" class="note-title marker" id="h-note-14" aria-level="5"><span>注記 3</span></div><p class="">This criterion applies to content that appears in addition to the triggering component itself. Since hidden components that are made visible on keyboard focus (such as links used to skip to another part of a page) do not present additional content they are not covered by this criterion.</p></div>
+     <div class="note" role="note" id="issue-container-generatedID-14"><div role="heading" class="note-title marker" id="h-note-14" aria-level="5"><span>注記 3</span></div><p class="">この達成基準は、トリガーとなるコンポーネント自体に加え表示されるコンテンツに対して適用される。隠されたコンポーネントがキーボードフォーカスによって可視化される (例えば、ページの別の部分にスキップするためのリンク) 場合は「追加コンテンツ」を提示するわけではないので、この達成基準が扱う対象ではない。</p></div>
```

Comment:
in addition toを「追加で」としてみました
hiddenは「非表示」でよいのではないでしょうか…（後でまとめて訳語検討でしょうが）
coverは「〈規則などが〉…に適用される」とあるのでそのようにしてみました。

```suggestion
     <div class="note" role="note" id="issue-container-generatedID-14"><div role="heading" class="note-title marker" id="h-note-14" aria-level="5"><span>注記 3</span></div><p class="">この達成基準は、トリガーとなるコンポーネント自体に追加で表示されるコンテンツに対して適用される。キーボードフォーカスによって可視化される非表示のコンポーネント (例えば、ページの別の部分にスキップするためのリンク) は追加コンテンツを提示しないため、この達成基準には適用されない。</p></div>
```